File: sdkwork-app-sdk-csharp/Api/StatisticsApi.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using App.Http;
using App.Models;

namespace App.Api
{
    public class StatisticsApi
    {
        private readonly HttpClient _client;

        public StatisticsApi(HttpClient client)
        {
            _client = client;
        }

        /// <summary>
        /// 获取组织统计
        /// </summary>
        public async Task<PlusApiResultOrganizationStatisticsVO?> GetOrganizationAsync()
        {
            return await _client.GetAsync<PlusApiResultOrganizationStatisticsVO>(ApiPaths.AppPath("/organization/statistics"));
        }
    }
}

File: sdkwork-app-sdk-swift/Sources/API/StatisticsApi.swift
import Foundation

public class StatisticsApi {
    private let client: HttpClient
    
    public init(client: HttpClient) {
        self.client = client
    }

    /// 获取组织统计
    public func getOrganization() async throws -> PlusApiResultOrganizationStatisticsVO? {
        let response = try await client.get(ApiPaths.appPath("/organization/statistics"))
        return response as? PlusApiResultOrganizationStatisticsVO
    }
}
